samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AuthorMessages_ca.properties

```diff
@@ -89,6 +89,7 @@ loading=Carregant preguntes...
 # CALCULATED_QUESTION
 calculated_question=Pregunta calculada
 calc_question_var_label=Variables
+calc_question_globalvar_label=Variables globals
```

Comment:
All the new variables at the end of the file with a comment.

Reply:
Added at the end